### DISCUSSION TITLE
deno runnable tasks issue

### DISCUSSION THREAD

**Discussion by sigmaSd** [zed-industries/zed#11586] [Feature Requests]
I tried adding runnable tasks to deno https://github.com/sigmaSd/zed-deno

- normal tasks works
- for tasks that require a tag, the marker shows up in the editor but when I press it nothing happens
![image](https://github.com/zed-industries/zed/assets/22427111/00f81cf0-b947-4190-988c-90dedcaaf43c)

**Comment by sigmaSd** [comment]
Also I lose syntax highlight

**Reply by RemcoSmitsDev** [reply]
If you need some help, let me know!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
